app/views/admin/site_customization/pages/edit.html.erb

```diff
@@ -1,11 +1,11 @@
-<% provide :title do %>
-  <%= t("admin.header.title") %> - <%= t("admin.menu.site_customization.pages") %> - <%= @page.title %>
+<% provide :slug do %>
+  <%= t("admin.header.title") %> - <%= t("admin.menu.site_customization.pages") %> - <%= @page.slug %>
 <% end %>
 
 <%= back_link_to admin_site_customization_pages_path %>
 
 <div class="small-12 column">
-  <h2 class="inline-block"><%= t("admin.site_customization.pages.edit.title", page_title: @page.title) %></h2>
+  <h2 class="inline-block"><%= t("admin.site_customization.pages.edit.title", page_title: @page.slug) %></h2>
```

Comment:
Similar comment as the previous one. Should admins see "Editing My Page" or "Editing my-page"? Maybe @decabeza can tell better.

Reply:
Hi, @papayalabs and @javierm! I think it's better always to use the **page title**. Usually, the slugs will have a dash or underscore and it'll a little more uncomprehensive. 😌 

I think is easier to read 
`Administration - Editing My custom page` 
instead of 
`Administration - Editing my-custom-page`